sphinx/source/docs/user_guide/examples/styling_linear_mappers.py

```diff
@@ -17,7 +17,7 @@
 
 p.circle(x='x', y='y', line_color=mapper,color=mapper, fill_alpha=1, size=12, source=source)
 
-color_bar = ColorBar(color_mapper=mapper['transform'], width=8,  location=(0,0))
+color_bar = ColorBar(color_mapper=mapper['transform'], width=8)
```

Comment:
Forgot about this bit of ugliness. We should probably add an `accepts` to adapt the `linear_cmap` dict and pull out the transform from it automatically (not necessarily in this PR)

Reply:
That's an option. I would also (or at least) use dataclasses to turn `mapper['transform']` into `mapper.transform`. That would help with readability a lot.

Reply:
That would also be a great improvement